app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :likes
+  has_many :comments
```

Comment:
- [ ] As requested in the project requirements `author_id` column should be the foreign_key for the user's table.

```ruby
 has_many :posts, foreign_key: 'author_id'
 has_many :comments, foreign_key: 'author_id'
 has_many :likes, foreign_key: 'author_id'